README.md

```diff
@@ -137,7 +140,7 @@ public class Meals : Table {
 import SwiftKuery
 import SwiftKueryPostgreSQL
 ```
-6. Inside the `App` class create a `Meals` table object by inserting the line:
+6. Inside the `App` class create an instance of the  `Meals` table object by inserting the following line:
```

Comment:
Line 150 needs changing (sorry it isn't this line)... it is "We will now connect to our server to the PostgreSQL database. " there are too many to's!